tests/testdata/pgkit_test_db.sql

```diff
@@ -21,8 +21,9 @@ CREATE TABLE logs (
 
 CREATE TABLE stats (
   id SERIAL PRIMARY KEY,
-  key VARCHAR(80),
-  big_num NUMERIC(78,0) -- representing a *big.Int runtime type
+  key VARCHAR(80) UNIQUE,
+  big_num NUMERIC(78,0) NOT NULL, -- representing a big.Int runtime type
+  rating NUMERIC(78,0) NULL -- representing a *big.Int runtime type
```

Comment:
let update comment to `-- representing a nullable big.Int runtime type`

Reply:
good finding, fixed